UnityProject/Assets/Scripts/Systems/Inventory/Inventory.cs

```diff
@@ -106,7 +106,9 @@ public static bool ServerDespawn(GameObject objectInSlot)
 		if (pu == null || pu.ItemSlot == null)
 		{
 			_ = Despawn.ServerSingle(objectInSlot);
+			return true;
```

Comment:
Probably a good idea to await and return `Despawn.ServerSingle()`'s successful state.